include/seadsa/sea_dsa.h

```diff
@@ -15,15 +15,21 @@ extern void sea_dsa_set_read(const void *p);
 extern void sea_dsa_set_modified(const void *p);
 // sea-dsa will mark the node pointed to by p as ptr to int (2)
 extern void sea_dsa_set_ptrtoint(const void *p);
+// sea-dsa will mark the node pointed to by p as ptr to int (P)
+extern void sea_dsa_set_inttoptr(const void *p);
+// sea-dsa will mark the node pointed to by p as heap memory (H)
+extern void sea_dsa_set_heap(const void *p);
+// sea-dsa will mark the node pointed to by p as stack memory (S)
+extern void sea_dsa_set_alloca(const void *p);
 // sea-dsa will collapse the argument's cell
 extern void sea_dsa_collapse(const void *p);
 // sea-dsa will return a fresh memory object
-extern void* sea_dsa_new();
+extern void *sea_dsa_new();
```

Comment:
mark with `__attribute__((malloc))` so that it plays nicely with other parts of LLVM.
Also a good idea to add attributes to other functions. `malloc` is the one attribute I used yesterday so that's why I remember it.